openupgrade_scripts/scripts/website_sale/17.0.1.1/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,147 @@
+---Models in module 'website_sale'---
+---Fields in module 'website_sale'---
+website_sale / product.document         / shown_on_product_page (boolean): NEW
+website_sale / product.product          / rating_ids (one2many)         : is now stored
+website_sale / product.product          / rating_ids (one2many)         : module is now 'rating' ('website_sale')
+website_sale / product.product          / rating_ids (one2many)         : not related anymore
+website_sale / product.product          / ribbon_id (many2one)          : NEW relation: product.ribbon
+website_sale / product.ribbon           / product_tag_ids (one2many)    : DEL relation: product.tag
+website_sale / product.tag              / image (binary)                : NEW attachment: True
+website_sale / product.tag              / ribbon_id (many2one)          : DEL relation: product.ribbon
+website_sale / product.tag              / visible_on_ecommerce (boolean): NEW hasdefault: default
```

Comment:
not sure if it is better to do this by query in pre-migration.

Reply:
You mean filling in the default instead of the ORM doing it?

Reply:
Yes.

Reply:
More than wondering about performance, in this case I think we should set this to False for not changing the behavior of the website appearing in this version "new things" in the website.

Reply:
done